From 4da2fb246951f5af917f80a4d7e61d670fa198fa Mon Sep 17 00:00:00 2001
From: Mohit Sabharwal <mohit@cloudera.com>
Date: Thu, 11 Feb 2016 16:27:33 -0800
Subject: [PATCH 0496/1164] CDH-37158 : HIVE-13045 : Move guava dependency
 back to 14.

HIVE-12952 removed usage of EvictingQueue, so we don't need to up
dependency to guava 15 at this point - avoid version related conflicts
with clients if we can avoid it.

Change-Id: I04032bdb7bcedf09636eafa67306e12c2b582e13
---
 pom.xml                                            |    2 +-
 .../service/cli/operation/OperationManager.java    |    1 -
 2 files changed, 1 insertion(+), 2 deletions(-)

diff --git a/pom.xml b/pom.xml
index cbc96dd..4cc2193 100644
--- a/pom.xml
+++ b/pom.xml
@@ -122,7 +122,7 @@
     <commons-dbcp.version>1.4</commons-dbcp.version>
     <derby.version>10.11.1.1</derby.version>
     <metrics-core.version>${cdh.metrics-core.version}</metrics-core.version>
-    <guava.version>15.0</guava.version>
+    <guava.version>14.0.1</guava.version>
     <groovy.version>2.4.4</groovy.version>
     <hadoop-20S.version>${cdh.mr1.version}</hadoop-20S.version>
     <hadoop-23.version>${cdh.hadoop.version}</hadoop-23.version>
diff --git a/service/src/java/org/apache/hive/service/cli/operation/OperationManager.java b/service/src/java/org/apache/hive/service/cli/operation/OperationManager.java
index bbb6bdf..b1adadd 100644
--- a/service/src/java/org/apache/hive/service/cli/operation/OperationManager.java
+++ b/service/src/java/org/apache/hive/service/cli/operation/OperationManager.java
@@ -33,7 +33,6 @@
 
 import org.apache.commons.logging.Log;
 import org.apache.commons.logging.LogFactory;
-import com.google.common.collect.EvictingQueue;
 import org.apache.hadoop.hive.common.metrics.common.Metrics;
 import org.apache.hadoop.hive.common.metrics.common.MetricsConstant;
 import org.apache.hadoop.hive.common.metrics.common.MetricsFactory;
-- 
1.7.9.5

